### DISCUSSION TITLE
New User Setup - Linux Recommendations (Raspberry Pi Option?)

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#5121] [Q&A]
I am keen to get a local setup of InvenTree running, but I have limited (none!) Linux OS knowledge.

I initially considered a hosted solution, but I would prefer a self-hosted solution running in the shop. The setup is for a small shop running inFlow On-Premise, but they are moving to a cloud-based (charge per month) model, so time to look for a setup I can manage. Have about 6,000 inventory items with multiple storage locations. The user count to access the data would be single user, so not a very heavily used install.

Looking for suggestions on a Linux version to run that is also good for a first-time user that would be suitable for an InvenTree installation. Someone had suggested a Raspberry Pi Model 4 8/16GB model. From what I can see, it seems to run the Pi OS, which is a Debian derivative. Is a Raspberry Pi a suitable host for running InvenTree?

If not, what other alternatives are there? I tried installing Debian 11 64-bit, but found that the installation process of InvenTree failed as the Docker was not validated/approved. It might be an easy process to override the warning, but being illiterate about Linux does not help.

Feedback would be appreciated. I like the idea of a small form factor like the Raspberry Pi, but not sure it has sufficient resources (or compatibility to host a suitable installation.

Thanks..

**Comment by SergeoLacruz** [comment]
In installed it on a RPi 3 some months ago and it worked. But I found this setup  too slow. No fun. May be Pi4 is better. 
I changed to an old HP Elitedesk running latest LTS Ubuntu. No problems with that.

**Reply by imager2019** [reply]
I thought the RPi idea was pushing it. 
In the meantime, I have acquired a Lenovo ThinkCenter M910 and will give that a try.
Thanks..
